patches/components-invalidation-impl-push_client_channel.cc.patch

```diff
@@ -1,20 +0,0 @@
-diff --git a/components/invalidation/impl/push_client_channel.cc b/components/invalidation/impl/push_client_channel.cc
-index e5de7eb5c1c8fcef6970a20825780ce43bce58a5..b54988be5c4c1ec11ecd5635c01c9e0156b5514f 100644
---- a/components/invalidation/impl/push_client_channel.cc
-+++ b/components/invalidation/impl/push_client_channel.cc
-@@ -28,6 +28,7 @@ PushClientChannel::PushClientChannel(
-     : push_client_(std::move(push_client)),
-       scheduling_hash_(0),
-       sent_messages_count_(0) {
-+  return; // feature disabled in Brave
```

Comment:
@pilgrim-brave since this has been removed in Chromium, do we need to do anything in the "new infrastructure" (fcm_invalidation)?

Reply:
fcm_invalidation is already patched. https://github.com/brave/brave-core/blob/master/patches/components-invalidation-impl-fcm_network_handler.cc.patch